src/lib/hooks/useCurrency.ts

```diff
@@ -121,7 +122,8 @@ export function useTokenFromActiveNetwork(tokenAddress: string | undefined): Tok
     () => decimals.loading || symbol.loading || tokenName.loading,
     [decimals.loading, symbol.loading, tokenName.loading]
   )
-  const parsedDecimals = useMemo(() => decimals.result?.[0], [decimals.result])
+  const parsedDecimals = useMemo(() => decimals?.result?.[0] ?? DEFAULT_ERC20_DECIMALS, [decimals.result])
```

Comment:
Using `??` means that `parsedDecimals` will never be `null | undefined`, so L140 is dead code.

Reply:
tru, thanks done.